### ISSUE TITLE
[Question] does vex support streaming uploads

### ISSUE THREAD

**Issue by ChewingBever** [nedpals/vex#47]
Hi everyone

So I want to re-make one of my Python projects in V, and for that I have one big requirement: I need to be able to stream data to the server using a PUT or a POST request. The application I'm building will be the upload point for CI output that could reach a gigabyte in size, and I'd like to be able to stream multiple of these files at once to the server without having to completely load the file into memory. I tried this using vweb & discovered that vweb *does* try to load the entire file into memory, so I'd like to try vex, but only if it does support this feature.

tl;dr would vex load an entire file into memory when it's sent using a PUT request?

Thanks in advance!
